<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>835</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="GraphicsView" name="graphicsView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAddLine">
    <property name="title">
     <string>Lines</string>
    </property>
    <addaction name="actionAddLine"/>
   </widget>
   <addaction name="menuAddLine"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="drawToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionSelectGraphObject"/>
   <addaction name="actionAddLine"/>
   <addaction name="actionAddCircle"/>
   <addaction name="actionAddRectangle"/>
   <addaction name="actionAddText"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetTree">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="GraphicsTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetCreator">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockCreatortContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_selection_mode">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>201</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <attribute name="label">
         <string>Selection mode</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>Rectangle</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Line</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_snapmode">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>201</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Snapping</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5"/>
       </widget>
       <widget class="QWidget" name="page_properties">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>201</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Pen</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTreeWidget" name="tree_properties">
           <property name="columnCount">
            <number>2</number>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">2</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>201</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Geometry</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="isWrapping" stdset="0">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>test1</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
           </item>
           <item>
            <property name="text">
             <string>test2</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="viewToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScalePlus"/>
   <addaction name="actionScaleMinus"/>
  </widget>
  <action name="actionAddLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="second.qrc">
     <normaloff>:/line.png</normaloff>:/line.png</iconset>
   </property>
   <property name="text">
    <string>addLine</string>
   </property>
  </action>
  <action name="actionSelectGraphObject">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="second.qrc">
     <normaloff>:/select_arrow.png</normaloff>:/select_arrow.png</iconset>
   </property>
   <property name="text">
    <string>selectGraphObject</string>
   </property>
  </action>
  <action name="actionAddCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="second.qrc">
     <normaloff>:/circle.png</normaloff>:/circle.png</iconset>
   </property>
   <property name="text">
    <string>addCircle</string>
   </property>
  </action>
  <action name="actionAddRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../../projects/second/trunk/src/second.qrc">
     <normaloff>:/rectangle.png</normaloff>:/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>addRectangle</string>
   </property>
  </action>
  <action name="actionScalePlus">
   <property name="icon">
    <iconset resource="../../../../../projects/second/trunk/src/second.qrc">
     <normaloff>:/lupe_plus.png</normaloff>:/lupe_plus.png</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionScaleMinus">
   <property name="icon">
    <iconset resource="../../../../../projects/second/trunk/src/second.qrc">
     <normaloff>:/lupe_minus.png</normaloff>:/lupe_minus.png</iconset>
   </property>
   <property name="text">
    <string>ScaleMinus</string>
   </property>
  </action>
  <action name="actionAddText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../../projects/second/trunk/src/second.qrc">
     <normaloff>:/text.png</normaloff>:/text.png</iconset>
   </property>
   <property name="text">
    <string>addText</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>GraphicsTreeView</class>
   <extends>QTreeView</extends>
   <header>graphicstreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="second.qrc"/>
 </resources>
 <connections/>
</ui>
